extended/src/test/java/apoc/es/ElasticSearchTest.java

```diff
@@ -338,4 +424,20 @@ public void testGetQueryUrlShouldNotHaveTrailingQuestionMarkIfQueryIsEmpty() {
         assertNotEquals(queryUrl, es.getQueryUrl(host, index, type, id, new HashMap<String, String>()));
         assertTrue(!es.getQueryUrl(host, index, type, id, new HashMap<String, String>()).endsWith("?"));
     }
+
+    private static Consumer<Map<String, Object>> commonEsGetConsumer() {
+        return r -> {
+            Object name = extractValueFromResponse(r, "$._source.name");
+            assertEquals("Neo4j", name);
+        };
+    }
+
+    private static Consumer<Map<String, Object>> commonEsStatsConsumer() {
+        return r -> {
+            assertNotNull(r.get("value"));
+
+            Object numOfDocs = extractValueFromResponse(r, "$._all.total.docs.count");
+            assertNotEquals(0, numOfDocs);
```

Comment:
Perché mettiamo not equals? Perché la count non è esatta? In caso scriviamolo e mettiamo un link alla documentazione di elastic

Reply:
Ho creato il metodo common prendendo le assertion da qui https://github.com/neo4j-contrib/neo4j-apoc-procedures/commit/90d7870e854d1a817b4b1fa4a91656bc86f614b1. 
Ma credo fosse un refuso, adesso ho messo `assertEquals(3, numOfDocs)`